### ISSUE TITLE
Show battery/ pressure warning in frontend

### ISSUE THREAD

**Issue by UsamaEquinorAFK** [equinor/flotilla#722]
**Describe the new feature you would like to see**
Show warning based on the current threshold values set for the robot model. 

**Describe the solution you'd like**
NA

**How will this feature affect the current Threat Model?**
NA

**Comment by UsamaEquinorAFK** [comment]
Its blocked by #721

**Comment by aestene** [comment]
This issue has automatically been marked as stale as there has been no activity for 60 days.

**Comment by anetteu** [comment]
Adding this back to robotics as a new issue to trigger a  prioritization during planning. Why is this no longer on the board?

**Comment by UsamaEquinorAFK** [comment]
We should discuss this with ux as well @mariannewiggen

**Comment by oysand** [comment]
This is an extension of #893

**Comment by aestene** [comment]
This issue has automatically been marked as stale as there has been no activity for 60 days.

**Comment by aestene** [comment]
Closing as this is covered more extensively by #893
